### PR TITLE
📖 Start v020 user guide

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR introduces a non-legacy user guide. Much more needs to be done, but this adds a few bits of information that we urgently want to get out.

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
/cc @pdettori 
/cc @francostellari 
/cc @ezrasilvera 
/cc @nirrozenbaum 
/cc @KPRoche 
@eaepstein

File: docs/content/v0.20/user-guide.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Using an existing Kind cluster as the hosting cluster
+
+The examples say to create a Kind cluster for hosting using the following command.
+
+```shell
+kflex init --create-kind
+```
+
+To use a pre-existing Kind cluster instead, make sure that your current kubeconfig context is for accessing that cluster and issue the following command.
```
Comment (francostellari) [inline docs/content/v0.20/user-guide.md:15]: The pre-existing kind cluster has 2 requirements
1. TLS passthrough
2. additional port 9443

Comment (MikeSpreitzer) [reply docs/content/v0.20/user-guide.md:17]: Revised in the force-push to 78693606a

Comment (MikeSpreitzer) [reply docs/content/v0.20/user-guide.md:17]: The force-push to a6d3e703a adds one clarification.

Comment (pdettori) [reply docs/content/v0.20/user-guide.md:17]: note that you could use a port different from 9443. Different port can be specified by the -p option in `kflex init`
See 
```
Usage:
  kflex init [flags]

Flags:
  -s, --chatty-status       chatty status indicator (default true)
  -c, --create-kind         Create and configure a kind cluster for installing Kubeflex
  -d, --domain string       domain for FQDN (default "localtest.me")
  -p, --externalPort int    external port used by ingress (default 9443)
  -h, --help                help for init
  -k, --kubeconfig string   path to kubeconfig file
  -v, --verbosity int       log level
 ```

Comment (pdettori) [reply docs/content/v0.20/user-guide.md:17]: anyway, this could be too much detail at this level

**Review by pdettori — APPROVED** [review]
/lgtm

**Pull request merged**